front/src/common/Pathfinding/pathfinding.scss

```diff
@@ -0,0 +1,70 @@
+.pathfinding-main-container {
+  margin-left: 1.5rem;
+
+  .pathfinding-subsection {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .pathfinding-done {
+    @extend .pathfinding-subsection;
+    
+    .title {
+      color: green;
+    }
+
+    .details {
+      font-weight: 400;
+    }
+  }
+
+  .pathfinding-error {
+    @extend .pathfinding-subsection;
+    color: red;
+  }
+
+  .missing-params {
+    @extend .pathfinding-subsection;
+    color: #ffb612;
+  }
+
+  .pathfinding-in-progress {
+    border-radius: 4px;
+    position: absolute;
+    top: 0;
+    left: 0;
+    min-width: 100%;
+    min-height: 100%;
+    background-color: rgba(0,0,0,0.1);
+    z-index: 2;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    .pathfinding-in-progress-card {
+      width: 50%;
+      border-radius: 4px;
+      padding: 1rem 1.5rem;
+      background: white;
```

Comment:
please use Bootstrap's color with var(--white) for example

Reply:
done